### ISSUE TITLE
iOS streaming does not refresh on value change

### ISSUE THREAD

**Issue by Titozzz** [launchdarkly/ios-client#201]
**Describe the bug**
When updating a variation value, the new value does not trigger a refresh.

**To reproduce**
Update a variation value, your event listener does not get triggered
I think the issue comes from here: https://github.com/launchdarkly/ios-client-sdk/blob/v4/LaunchDarkly/LaunchDarkly/Service%20Objects/FlagChangeNotifier.swift#L161

**Expected behavior**
Same as on android, value updates should trigger the event listener.

**SDK version**
I'm using react-native sdk 2.0.3 with iOS SDK 4.3.0

**Comment by Titozzz** [comment]
Meanwhile I know the fix is not good but kindof working, I did this:
```
return !(oldFeatureFlag.matchesVariation(newFeatureFlag) && oldFeatureFlag.value as? String == newFeatureFlag.value as? String)
```

**Comment by torchhound** [comment]
Hi @Titozzz the React Native SDK version 2.0.3 is only compatible with iOS SDK version 4.1.2. The use of 4.3.0 is undefined behavior. Please use the correct version and see if that fixes your issue.

**Comment by Titozzz** [comment]
@torchhound it does not, I've tried both versions, hence the fix I proposed above..

**Comment by torchhound** [comment]
@Titozzz  thank you for reporting this issue, it looks like this is a bug. We will remediate and release a new version soon. Filed as 58261.

**Comment by torchhound** [comment]
Hi @Titozzz this bug was just fixed in version 4.3.2. Thanks again for reporting it!

**Comment by Titozzz** [comment]
Thank you very much.
